docs/notifications.json

```
@@ -13,5 +13,10 @@
     "id": 71,
     "title": "<b>A major update is coming for MUI X—and you can get involved</b>",
     "text": "Check out the <a style=\"color: inherit;\" data-ga-event-category=\"Blog\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x-v6-alpha-zero\" href=\"/blog/mui-x-v6-alpha-zero/\">blog post</a> for details, and let us know what you want to see in MUI X v6."
+  },
+  {
+    "id": 72,
+    "title": "<b>The MUI Developer Survey is live!</b>",
```

Would this work for the standard capitalization?

```suggestion
    "title": "<b>The MUI Developer survey is live!</b>",
```